src/modules/getScores.js

```diff
@@ -0,0 +1,8 @@
+const getScore = async () => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/fcVHLAyQwdIZeixcHMPy/scores')
+    .then((response) => response.json())
+    .then((data) => localStorage.setItem('result', JSON.stringify(data.result)))
+    .catch((error) => error);
+};
+
```

Comment:
- [ ] This data is not getting fetched and I don't know why you are setting it to local storage. It can be always fetched from the API and rendered on the UI.